src/amo/components/Notice/styles.scss

```diff
@@ -23,6 +23,7 @@
 }
 
 .Notice-icon {
+  align-self: center;
```

Comment:
Do we want it to be vertically aligned? I don't really like how it looks when the notice contains a multiline string.


<img width="368" alt="Screen Shot 2021-05-07 at 10 22 45" src="https://user-images.githubusercontent.com/217628/117420771-2d5ecf80-af1e-11eb-933a-eb29c238cdca.png">

Reply:
I think we could increase the margin a bit. For example here is how it looks with `margin: 7px;`:

<img width="364" alt="Screen Shot 2021-05-07 at 10 28 35" src="https://user-images.githubusercontent.com/217628/117421617-fc32cf00-af1e-11eb-8002-3e27f9bf5467.png">

I think that's what we had originally.

Reply:
@willdurand I agree it looks better with the icon aligned with the first line of text, but how did you get that to work?

Reply:
I changed the margin value